content/python/concepts/built-in-functions/terms/memoryview/memoryview.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: 'memoryview()'
+Description: 'Creates a 'memory view' object that allows Python code to access the internal data of an object's buffer without making a copy of it'
+Subjects:
+  - 'Data Types'
+  - 'Computer Science'
+  - 'Built-in Functions'
+Tags:
+
+CatalogContent:
+---
+
+Creates a `memory view` object that allows Python code to access the internal data of an object's buffer without making a copy of it.
+
+## Syntax
+
+```python
```

Comment:
The syntax section should be something more like this:

````
```pseudo
result = memoryview(obj)
```

Where `obj`  is the object to create a `memoryview` of, and `result` is the `memoryview` created.
````